src/escape_chars.c

```diff
@@ -85,6 +88,13 @@ SEXP C_escape_chars_one(SEXP x) {
         new_p++;
         *new_p = '\0';
         break;
+      case '/':
+        if(escape_solidus){
+          *new_p = '\\';
+          new_p++;
+          *new_p = '/';
+          break;
```

Comment:
Seems a little risky to put the `break` in the `if`. If later you add another case after `/`,  the `/` case will fall through to it when `esacpe_solidus` is false.